drivers/volume/volume.go

```diff
@@ -199,11 +200,17 @@ type Driver interface {
 	// GetLicenseSummary returns the activated license SKU and Features
 	GetLicenseSummary() (LicenseSummary, error)
 
-	//SetClusterOpts sets cluster options
+	// SetClusterOpts sets cluster options
 	SetClusterOpts(n node.Node, rtOpts map[string]string) error
 
-	//ToggleCallHome toggles Call-home
+	// ToggleCallHome toggles Call-home
 	ToggleCallHome(n node.Node, enabled bool) error
+
+	// ExpandPool resizes a pool of a given ID
+	ExpandPool(poolUID string, operation api.SdkStoragePool_ResizeOperationType, size uint64) error
```

Comment:
Can you shrink a pool? If yes, may be rename this to ResizePool. Otherwise, it's fine.

Reply:
nope, we can't shrink a pool, only expand